gnomad_qc/v4/annotations/generate_variant_qc_annotations.py

```diff
@@ -133,13 +181,14 @@ def main(args):
     run_vep = args.run_vep
     overwrite = args.overwrite
     resources = get_variant_qc_annotation_resources(test=test, overwrite=overwrite)
```

Comment:
I know this is outside the scope of this immediate PR, but the naming of this was a bit confusing (for me!) with both gnomad.resources and gnomad_qc.v4.resources already existing. is it possible to read in as 'vqc_resources' or something a bit more clear .